src/panorama/VideoPanorama.js

```diff
@@ -64,6 +75,9 @@
 		this.videoElement.muted = options.muted || false;
 		this.videoElement.loop = ( options.loop !== undefined ) ? options.loop : true;
 		this.videoElement.autoplay = ( options.autoplay !== undefined ) ? options.autoplay : false;
+		this.videoElement.crossOrigin = ( options.crossOrigin !== undefined ) ? options.crossOrigin : "anonymous";
+		this.videoElement.setAttribute( "webkit-playsinline", ( options["webkit-playsinline"] !== undefined ) ? options["webkit-playsinline"] : "" );
+		this.videoElement.setAttribute( "webkit-playsinline", ( options["webkit-playsinline"] !== undefined ) ? options["webkit-playsinline"] : "" );
```

Comment:
Seems to be duplicate with above line [L79](https://github.com/pchen66/panolens.js/pull/34/commits/c3c75a70be0ac8eddd696cd44531b26f79e597db#diff-e120260d5b4174e58f42b55e128aa2c7R79)